motor/scripts/motormotioncommands.py

```diff
@@ -0,0 +1,60 @@
+import RPi.GPIO as GPIO
+'''
+
+#Standard Pin Definitions for motor
+RIGHT1	=	17
+RIGHT2	=	18
+LEFT1	=	22
+LEFT2	=	23
+
+'''
+def initmotorpins(RIGHT1 = 17, RIGHT2 = 18, LEFT1 = 22, LEFT2 = 23):
+	#Set pins as outputs
+	GPIO.setup(RIGHT1, GPIO.OUT)
+	GPIO.setup(RIGHT2, GPIO.OUT)
+	GPIO.setup(LEFT1, GPIO.OUT)
+	GPIO.setup(LEFT2, GPIO.OUT)
+	#Set all pins to LOW
+	GPIO.output(RIGHT1, GPIO.LOW)
+	GPIO.output(RIGHT2, GPIO.LOW)
+	GPIO.output(LEFT1, GPIO.LOW)
+	GPIO.output(LEFT2, GPIO.LOW)
+	
+
+def moveforward():
+	# Note: Output arrangement is important to make sure there is no jitter in the motors when moving Forwards/Backwards
+	# TODO: explain the reason for the arrangement in detail in the documentation. 
+	GPIO.output(RIGHT1, GPIO.HIGH)
+	GPIO.output(LEFT1, GPIO.HIGH)
+	GPIO.output(RIGHT2, GPIO.LOW)
+	GPIO.output(LEFT2, GPIO.LOW)
+
+def movebackwards():
+	# Note: Output arrangement is important to make sure there is no jitter in the motors when moving Forwards/Backwards
+	# TODO: explain the reason for the arrangement in detail in the documentation. 
+	GPIO.output(RIGHT1,GPIO.LOW)
+	GPIO.output(LEFT1, GPIO.LOW)
+	GPIO.output(RIGHT2, GPIO.HIGH)
+	GPIO.output(LEFT2, GPIO.HIGH)
+
+def turnright():
+	# Note: Output arrangement is important to make sure there is no jitter in the motors when turning Right/Left
+	# TODO: explain the reason for the arrangement in detail in the documentation. 
+	GPIO.output(LEFT1, GPIO.HIGH)
+	GPIO.output(RIGHT2, GPIO.LOW)
```

Comment:
The vehicle is tank shaped, I think it's better to set right motor to reverse to make a circle turn right.
Same goes for turnleft()

Reply:
That's a good point! This is resolved now.

Reply:
I meant moving right motor in reverse while still moving the left forward :)   but again, it is fine,, let's push this and we can change it in real time

Reply:
I think moving one motor in a certain direction and leaving the other one in place would provide a more predictable turning motion, where there will be a pivot point. If both moved, and for some reason there was a difference in speed then the vehicle wouldn't rotate in a circle.